apps/storybook/src/Button.stories.tsx

```diff
@@ -87,6 +88,69 @@ WithIcon.args = {
   startIcon: <SvgAdd />,
 };
 
+export const WithNotification: Story<ButtonProps> = (args) => {
+  return (
+    <div style={{ display: 'flex', gap: '10px' }}>
+      <Button
+        onClick={action('clicked')}
+        startIcon={
+          <NotificationMarker status='white'>
+            <SvgEmail />
+          </NotificationMarker>
+        }
+        styleType='high-visibility'
+        {...args}
+      >
+        {args.children}
+      </Button>
+      <Button
+        onClick={action('clicked')}
+        startIcon={
+          <NotificationMarker status='white'>
+            <SvgEmail />
+          </NotificationMarker>
+        }
+        styleType='cta'
+        {...args}
+      >
+        {args.children}
+      </Button>
+      <Button
+        onClick={action('clicked')}
+        startIcon={
+          <NotificationMarker>
+            <SvgEmail />
+          </NotificationMarker>
+        }
+        styleType='default'
+        {...args}
+      >
+        {args.children}
+      </Button>
+      <Button
+        onClick={action('clicked')}
+        startIcon={
+          <NotificationMarker>
+            <SvgEmail />
+          </NotificationMarker>
+        }
+        styleType='borderless'
+        {...args}
+      >
+        {args.children}
+      </Button>
+    </div>
+  );
+};
+
+WithNotification.args = {
+  children: 'Inbox',
+};
+
+WithIcon.argTypes = {
```

Comment:
I think this should be "WithNotification.argTypes"?

Reply:
**[DONE]**